planner/core/rule_inject_extra_projection.go

```diff
@@ -121,11 +126,10 @@ func injectProjBelowAgg(aggPlan PhysicalPlan, aggFuncs []*aggregation.AggFuncDes
 	}
 
 	child := aggPlan.Children()[0]
-	prop := aggPlan.GetChildReqProps(0).Clone()
-	proj := PhysicalProjection{
+	proj := PhysicalProjection{ // give a nil property since physical optimization has finished
 		Exprs:                projExprs,
 		AvoidColumnEvaluator: false,
-	}.Init(aggPlan.context(), child.statsInfo().ScaleByExpectCnt(prop.ExpectedCnt), prop)
+	}.Init(aggPlan.context(), child.statsInfo())
```

Comment:
I prefer the old ways. a nil property may confuses the users when they see the result of `explain`.